package.json

```diff
@@ -25,7 +25,7 @@
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "ember-cli-babel": "^6.0.0",
     "ember-cli-version-checker": "^2.0.0",
-    "ember-compatibility-helpers": "^0.1.0",
+    "ember-compatibility-helpers": "^0.1.2",
```

Comment:
`ember-compatibility-helpers` was also affected by a similar bug, bumped the version explicitly just in case.